core/src/test/scala/unit/kafka/admin/TopicCommandTest.scala

```diff
@@ -240,6 +240,17 @@ class TopicCommandTest extends ZooKeeperTestHarness with Logging with RackAwareT
     } catch {
       case _: Exception => // topic creation should fail due to the invalid config
     }
+
+    // try to create the topic with another invalid config
+    try {
+      val createOpts = new TopicCommandOptions(
```

Comment:
This tests the path through zookeeper, which is good. Could we also have an integration test for the AdminClient which uses the CreateTopic API?